scripts/setup-env-ubuntu.sh

```diff
@@ -10,22 +10,24 @@ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubun
 curl -fsSL https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
 curl -fsSL https://nvidia.github.io/nvidia-docker/ubuntu"$(lsb_release -rs)"/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
 
-curl -fsSL https://deb.nodesource.com/setup_11.x | sudo -E bash -
+curl -fsSL https://deb.nodesource.com/setup_12.x | sudo -E bash -
 
 sudo add-apt-repository -y ppa:deadsnakes/ppa
```

Comment:
Can't we get rid of this yet?

Reply:
I generally favor using the native version of python.  Not sure if centos's version is new enough but... in most other cases it seems advisable.

Or in other words, I am always in favor of doing the most native thing unless there's a compelling reason not to.

Reply:
I like the principle, but in this case I think the compelling reason to break it is there (I know I didn't give details before, sorry): we still declare support for Ubuntu 16.04, which ships with Python 3.5, which won't work (the harness requires 3.6). I'm really hoping we can drop that soon, but it hasn't happened yet. (I just put it as an item on the AG doc.) On the other end, we also can't use the default 3.8 of Arch or Ubuntu 20.04, since we list TensorFlow 1.14 as a requirement, which isn't `pip`-installable on 3.8.

Reply:
I'll buy that.

Reply:
Now that I think about it more, why would 16.04 be supported by these scripts?  These scripts are for developers, right?  We should just not support dev environments that old.